### DISCUSSION TITLE
How to Connect Hot Reload via USB Cable in Android Physical Devices ?

### DISCUSSION THREAD

**Discussion by RedEye-Developers** [unoplatform/uno#18820] [Q&A]
In My Android Mobile Wireless Debugging in Disable and I Connected My Android Physical Devices using USB Cable to My PC Still if i Disconnect My Wifi Connection the Hot Reload was Disconnecting and After adding To Many UI Element in Page the Hot Reload was Too Slow have anyway To Improve the Hot Reload Performance in Physical Device ?

**Comment by jeromelaban** [comment]
Thanks for the report. Can you tell which IDE and OS you're using?

**Reply by RedEye-Developers** [reply]
Windows 10 Pro, JetBrains Rider!

**Reply by jeromelaban** [reply]
Thanks. In this case, Hot Reload needs the device to be connected to the same network than rider is using.

What's the issue that you're reporting? Is it that hot reload takes time to be applied on your device?

**Reply by RedEye-Developers** [reply]
yea it is taking long time after adding may UI Elements in Page!

it is Possible to Change Hot Reload Connection Via USB Cable ?

**Reply by jeromelaban** [reply]
Not at this time using Rider. We're investigating the ability to include Debugger channel support, but it's tied to Rider portions we do not have access to at this time.
